chronicles.nim

```diff
@@ -346,11 +346,11 @@ template log*(lineInfo: static InstInfo,
 template wrapSideEffects(debug: bool, body: untyped) {.inject.} =
   when debug:
     {.noSideEffect.}:
-      when (NimMajor, NimMinor) > (1, 6):
+      when (NimMajor, NimMinor) >= (1, 6):
```

Comment:
this warning has not been released yet - it means 1.6.10 for example fails with `invalid pragma: warning[BareExcept]:` - we can't put this check in until 1.6.12 is out